addons/mail/static/src/models/composer/composer.js

```diff
@@ -234,6 +234,22 @@ function factory(dependencies) {
             return [['replace', recipients]];
         }
 
+        /**
+         * Called when current partner is inserting some input in composer.
+         * Useful to notify current partner is currently typing something in the
+         * composer of this thread to all other members.
+         */
+        handleCurrentPartnerIsTyping() {
+            if (!this.thread) {
+                return;
+            }
+            if (this.thread.typingMembers.includes(this.env.messaging.currentPartner)) {
+                this.thread.refreshCurrentPartnerIsTyping();
+            } else {
+                this.thread.registerCurrentPartnerIsTyping();
+            }
+        }
```

Comment:
Is this just move for alphabetical? Confusing in current PR, I would avoid this diff in this already tricky bugfix.

Reply:
Yes it was for alphabetical. I reverted this change